Sources/PortalSwift/Portal.swift

```diff
@@ -204,6 +206,7 @@ public class Portal {
     // Initialize the Portal API
     let api = PortalApi(apiKey: apiKey, apiHost: apiHost, provider: self.provider, featureFlags: self.featureFlags)
     self.api = api
+    self.api.provider = self.provider
```

Comment:
The provider is already in the initializer above

Reply:
true! updated! ✅